mmd_tools/core/maya_utils.py

```
@@ -0,0 +1,1557 @@
+    """
+    try:
+        # オブジェクトのMObjectを取得
+        selection_list = om.MSelectionList()
+        selection_list.add(object_name)
+        node_obj = selection_list.getDependNode(0)
+        depend_fn = om.MFnDependencyNode(node_obj)
+
+        # プラグを取得（存在しない場合は例外が発生）
+        try:
+            plug = depend_fn.findPlug(attr_name, False)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)